src/common.jl

```diff
@@ -133,6 +133,26 @@ value_support(::Type{T}) where {T<:Distribution} = value_support(supertype(T))
 # to be decided: how to handle multivariate/matrixvariate distributions?
 Broadcast.broadcastable(d::UnivariateDistribution) = Ref(d)
 
+"""
+    minimum(d::Distribution)
+
+Return the minimum of the support of `d`.
+"""
+minimum(d::Distribution)
+
+"""
+    maximum(d::Distribution)
+
+Return the maximum of the support of `d`.
+"""
+maximum(d::Distribution)
```

Comment:
I am not familiar with this idiom. What does it mean?

Reply:
I haven't either, its from https://github.com/JuliaStats/Distributions.jl/pull/1319#discussion_r626645113

Reply:
and https://github.com/JuliaStats/Distributions.jl/pull/1319#discussion_r626651609